projects/ccd-case-ui-toolkit/src/lib/shared/directives/conditional-show/domain/conditional-show.model.ts

```diff
@@ -204,7 +204,7 @@ export class ShowCondition {
       if (conditionSeparaor === ShowCondition.CONDITION_NOT_EQUALS) {
         return this.checkValueNotEquals(expectedValue, currentValue);
       } else {
-        return currentValue == expectedValue || this.okIfBothEmpty(expectedValue, currentValue); // tslint:disable-line
+        return currentValue === undefined || currentValue == expectedValue || this.okIfBothEmpty(expectedValue, currentValue); // tslint:disable-line
```

Comment:
@Kartheek-S currentValue == expectedValue should be `===` instead of `==`?

Reply:
expectedValue might be a string or number i.e. sometimes we receive 2/'2'